### PR TITLE
Rework of locked field, using input field instead

### PR DESCRIPTION
**Pull request by alexwoll**
Used a input field instead of divs and stuff, works better with mobile design now.

skarmklipp 2016-03-17 10 23 15

### CODE REVIEW

**Comment by alexwoll** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;samI1Jhq&#x2F;1025-chrome-android-fake-email-field-overflowing"> [Chrome - Android] Fake email field overflowing</a>

File: src/mnd-bootstrap/css/_forms.scss [hunk 1]
```diff
@@ -216,17 +216,13 @@ textarea {
...
+.fake-field {
+  border: 0;
 
-  .fa-lock {
-    color: $text-color-light;
-    float: right;
-    padding-top: 3px;
+  &-text {
+    background: $ui-color-lighter;
+    border-radius: 6px;
```
Comment (mnd-hound) [inline src/mnd-bootstrap/css/_forms.scss]: Properties should be ordered background, border, border-radius, color

Comment (jorkas) [inline src/mnd-bootstrap/css/_forms.scss]: I think we already have a variable for `border-radius`?

**Comment by jorkas** [comment]
:shipit:

**Pull request merged**